wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,74 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+A user may get banned if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Sharing their account with other users.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What is considered cheating?
+The rules say that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player.
 
-This is a one-strike rule; cheaters will be **permanently banned.**
+This is a one-strike rule; cheaters will be *permanently banned*.
 
-I have been banned for cheating. I didn't cheat!
-------------------------------------------------
 
+## I have been banned for cheating. I didn't cheat!
 osu! uses an automated metrics combined with manual confirmation.
 
-The staff never places an incorrect ban for cheating. Before they do any actions, they make sure it first 100%.
+The team never places an incorrect ban for cheating. Before they take any actions, they make sure it is first 100%.
```

Comment:
"it is first 100%" sounds incomplete. How about "it is 100% sure you have cheated."

Reply:
Will change to: *Before they take any action, they make sure it is 100% that you cheated.*